app/Resources/views/my_page/my_page.html.twig

```diff
@@ -20,6 +20,8 @@
                     <div class="col-12 col-lg-6 mb-3 mb-lg-0">
                         {% include 'my_page/my_partners.html.twig' with {application: active_application} %}
                     </div>
+                {% elseif app.user.hasBeenAssistant %}
+                    <div class="col-lg-3"></div>
                 {% elseif not active_application.interview %}
```

Comment:
Kan disse kombineres? 
`elseif hasBeenAssistant or not active_application.interview`

Reply:
Ja, de kan kombineres, men jeg lot de være slik for at det skal samsvare med if-setnigen i https://github.com/vektorprogrammet/vektorprogrammet/blob/c620a2dec5131cc54711e8e41a11c175c309da28/src/AppBundle/Service/ApplicationManager.php

Reply:
For å gjøre det enklere å se hva hver if-blokk i `function getApplicationStatus` vil produsere i UIet

Reply:
Skjønner, ikke dumt :+1: